release: v1.55.0 (#6023)
apollographql/router: .changesets/fix_geal_remove_unused_fragments_from_filtered_queries.md

### Create valid documents with authorization filtering ([PR #5952](https://github.com/apollographql/router/pull/5952))

> This fixes the authorization plugin's query filtering to remove unused fragments and input arguments if the related parts of the query are removed. This was generating validation errors when the query went into query planning
> 

Reviewer: ```suggestion
This release fixes the authorization plugin's query filtering to remove unused fragments and input arguments if the related parts of the query are removed. Previously the plugin's query filtering generated validation errors when planning the query.

```
